snakemake/logging.py

```
@@ -305,378 +244,36 @@ def log_handler(self, msg):
-        Prints the output to the console.
-
-        Args:
-            msg (dict):     the log message dictionary
-        """
-        if self.is_quiet_about("all"):
-            # do not log anything
+def show_logs(logs):
+    """Helper method to show logs."""
+    for f in logs:
+        try:
+            content = open(f, "r").read()
```

Ruff

Use a context manager for opening files

(SIM115)